internal/pkg/manifest/backend_svc_test.go

```diff
@@ -1098,8 +1108,10 @@ func TestBackendService_ExposedPorts(t *testing.T) {
 					},
 					Sidecars: map[string]*SidecarConfig{
 						"xray": {
-							//Port:       aws.String("2000"),
-							Image:      aws.String("123456789012.dkr.ecr.us-east-2.amazonaws.com/xray-daemon"),
+							//Port: aws.String("2000"),
+							Image: Union[*string, SidecarImageConfig]{
+								Basic: aws.String("123456789012.dkr.ecr.us-east-2.amazonaws.com/xray-daemon"),
```

Comment:
For one of the test we can also try putting this URI inside `image.location` instead of `image`

Reply:
Yeah that'd be awesome! Perhaps not in `_ExposedPorts` tests but within the `transform_test`?

Reply:
Do we still need custom Transformer for mergo in `transform.go` and `transform_test` 🤔